server/routes/home/handlers/home.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Request, Response } from 'express'
+
+import HomeRoutes from './home'
+import AuthRole from '../../../enumeration/authRole'
+
+describe('Route Handlers - Home', () => {
+  const handler = new HomeRoutes()
+  let req: Request
+  let res: Response
+
+  beforeEach(() => {
+    res = {
+      render: jest.fn(),
+    } as unknown as Response
+  })
+
+  describe('GET', () => {
+    describe('For support admin role', () => {
+      it('With correct auth role', async () => {
+        req = getReqWithRolesAndSource([AuthRole.DECISION_MAKER], 'nomis')
+        await handler.GET(req, res)
+        expect(res.render).toHaveBeenCalledWith('pages/index', {
+          shouldShowSupportCard: false,
+        })
+      })
+
+      it('With wrong auth role', async () => {
+        req = getReqWithRolesAndSource([AuthRole.SUPPORT], 'nomis')
+        await handler.GET(req, res)
+        expect(res.render).toHaveBeenCalledWith('pages/index', {
+          shouldShowSupportCard: true,
+        })
+      })
+    })
+  })
+})
+
+const getReqWithRolesAndSource = (roles: string[], authSource: string): Request => {
+  return {
+    user: {
+      userRoles: roles,
+      authSource,
+    },
+  } as unknown as Request
+}
```

Comment:
Doesn't need to hold up this PR but we should probably agree on a testing strategy for this.. most projects use superagent rather than mocking... cvl is a bit of an outlier..